merci/evaluate.py

```diff
@@ -114,3 +115,46 @@ def evaluate(self) -> float:
 
         score = merci.measures.reliability_estimation(y_test, y_transductive)
         return score
+
+
+class DistributionShiftEvaluator(ModelEvaluator):
+    """
+    Evaluate robustness to distribution shift
+    """
+    def __init__(
+        self,
+        model: merci.types.Classifier,
+        train_dataset: merci.types.Dataset,
+        test_dataset: merci.types.Dataset,
+    ) -> None:
+        """
+        Initialize the transductive model evaluator.
+
+        :param model: Classification model to evaluate
+        :param train_dataset: A tuple containing input and target data for training
+        :param test_dataset: A tuple containing input and target data for testing
+        """
+        super().__init__(model, train_dataset, test_dataset)
+        self.thresholds = [0.75, 0.5, 0.25, 0.1]
```

Comment:
perhaps we could allow for optionally passing different thresholds, and setting this as a default?